README.md

```diff
@@ -1,18 +1,20 @@
-# Parallel Programming on Discovery
-This repository contains material for HPC Parallel Programming on Discovery
-training modules.
+# HPC & Parallel Computing
+This repository contains material for HPC and parallel computing on NU's high performance computing cluster training modules.
 
 ## What's in each training sub-folder?
-* Training PowerPoint slides
-* Exercise folders
+* Exercise_1 - OpenMP threads example
+* Exercise_2 - MPI with Python example (mpi4py)
+* Exercise_3 - Slurm job array example
+* Exercise_4 - Speedup, efficiency and weak scaling examples. 
```

Comment:
@marianalevineu, this is a nit-pick, but the exercise 4 and 5 lines end with a period whereas the exercise 1-3 lines do not.  Consider making them all consistent.

Reply:
thanks @dhml good point, I fixed it now as well.